web-admin/src/routes/[organization]/[project]/-/alerts/[alert]/open/+page.svelte

```diff
@@ -42,6 +43,9 @@
     },
     {
       exploreProtoState: alertSpec?.annotations?.web_open_state,
+      // When opening an alert from a link with token we should remove the filters from request.
+      // The filters are already baked into the token, each query will have them added in the backend.
```

Comment:
Out of curiosity, what would happen if we didn't remove the filters?

Reply:
It will end up applying the filters twice. Added a comment to clarify